server/fleet/software.go

```diff
@@ -103,7 +103,7 @@ type HostSoftwareEntry struct {
 	Software
 	// Where this software was installed on the host, value is derived from the
 	// host_software_installed_paths table.
-	InstalledPaths []string `json:"installed_paths,omitempty"`
```

Comment:
What is this change fixing?

Reply:
Sorry, converting to draft first. Still testing but got side tracked with release bugs

Reply:
3. This is for rendering a host software entry and to allow installed_paths to be empty and not be omitted if it's empty, wondering if that is a part of the issue with chrome extensions with no path (like on ChromeOS) not rendering as software

Reply:
OK, in any case the change should be innocuous :)